src/reseeding.rs

```diff
@@ -84,54 +80,11 @@ impl<R: Rng, Rsdr: Reseeder<R>> Rng for ReseedingRng<R, Rsdr> {
     }
 }
 
-impl<S, R: SeedableRng<S>, Rsdr: Reseeder<R> + Default>
-     SeedableRng<(Rsdr, S)> for ReseedingRng<R, Rsdr> {
```

Comment:
With an `[u8]` seed it is not possible to provide a reseeder. I just removed these function, we have a `new` function anyway. I dont know if it makes sense to add them back, with `ReseedWithNew` as default reseeder. That would only make it available with the `std` feature. I did not add `from_reseeder` as an alternative, I don't really like it.

Reply:
I'm fine with this as long as we maintain equivalent functionality in `StdRng` for now (we can also think about changing that later). It doesn't make much sense having reproducibility functionality in reseeding RNGs anyway.

Reply:
I don't completely follow what you mean, `StdRng` does not depend on `ReesedingRng`?

I am trying to think of a way to add it back, because I think `from_rng` can be a convenient way to initialize `ReesedingRng`. The `reseeder` is going to use `NewSeeded` usually anyway. But that is turning into quite some changes, better for another PR.

Reply:
For now, `thread_rng` reseeds periodically; as has been discussed the utility of this is questionable, especially since `OsRng` will not return week numbers on almost any OS.

I don't see any other use for `ReseedingRng`, so we could just remove it, other than a version specific to the requirements of `thread_rng`.

Reply:
I see some use in keeping `ReseedingRng`. It can provide a little extra sort of security. And we probably want to use it for the proposed fork protection. I think I see some possible use around cloning PRNG's, where one gets reseeded on first use after a clone. If we get the overhead down to ~~0%, I see no reason not to use it sometimes.

Reply:
Sure; I'm just saying don't worry about breaking the API here.